build/yamls/antrea-eks.yml

```diff
@@ -579,6 +579,18 @@ data:
 
     # Enable metrics exposure via Prometheus. Initializes Prometheus metrics listener.
     #enablePrometheusMetrics: false
+
+    # Provide flow collector address as string with format IP:port:L4(tcp or udp). This also enables flow exporter that sends IPFIX
+    # flow records of conntrack flows on OVS bridge. If no L4 transport proto is given, we consider tcp as default.
+    # Defaults to "".
+    #flowCollectorAddr: ""
+
+    # Provide flow exporter poll and export interval in format "0s:0s". This determines how often flow exporter polls connections
+    # in conntrack module and exports IPFIX flow records that are built from connection store.
+    # Any value in range [1s, ExportInterval(s)) for poll interval is acceptable.
+    # Any value in range (PollInterval(s), 600s] for export interval is acceptable.
+    # Defaults to "5s:60s". Follow the time units of duration.
+    #pollAndExportInterval: ""
```

Comment:
wouldn't it be more clear to have 2 parameters here: `flowPollInterval` and `flowExportInterval`?

Reply:
I changed the names to increase clarity. Since they are related to each other, I would like to keep it as a single parameter. Let me know if you think separate would be better.

Reply:
I do think `flowPollAndFlowExportIntervals` is a pretty awful name. I am not sure I see the rationale for bundling them as one configuration parameter when it is 2 independent entities (unlike say an IP address and a transport port number which together constitute an endpoint), so yes I do think it is better to separate them.

Reply:
so I see why you consider them related. But I still think we should separate them. Maybe a better solution would be to have a `pollInterval` parameter and a `exportFrequency` parameter, where `exportFrequency` is the number of poll intervals between exports. This way you have a natural way to guarantee that the export interval is a multiple of the poll interval.

Reply:
Agree it is a long name. The only rationale for keeping them together is that export interval has to be mutiple of poll interval. I see how it is uncommon and it is unnecessarily standing out as well among other config parameters. Will do the change.

Reply:
> so I see why you consider them related. But I still think we should separate them. Maybe a better solution would be to have a `pollInterval` parameter and a `exportFrequency` parameter, where `exportFrequency` is the number of poll intervals between exports. This way you have a natural way to guarantee that the export interval is a multiple of the poll interval.

This is a good suggestion.